near2eth/near2eth-block-relay/index.js

```diff
@@ -217,8 +262,35 @@ class Near2EthRelay {
         // console.log({bridgeState, currentBlockHash, lastBlock, replaceDuration}) // DEBUG
         if (bridgeState.currentHeight < lastBlock.inner_lite.height) {
           if (nextValidAt.isZero() || replaceDelay.cmpn(0) <= 0) {
+            // Serialize once here to avoid multiple 'borshify(...)' function calls
+            const blockCouple = {
+              lightClientBlock: lastBlock,
+              borshBlock: borshify(lastBlock)
+            }
+
+            // Describe the selection logic here so as not to complicate the reading of the code
+            if (nextBlockSelection.isEmpty()) {
+              console.log('The selection of the optimal block has begun.')
+              nextBlockSelection.set(blockCouple)
+            } else if (nextBlockSelection.isSuitable(blockCouple)) {
+              nextBlockSelection.set(blockCouple)
+            }
+
+            const selectDelay = selectDuration
+              .add(web3.utils.toBN(nextBlockSelection.startedAt))
+              .sub(web3.utils.toBN(lastBlock.inner_lite.timestamp))
+            if (selectDelay.cmpn(0) > 0) {
+              const selectDelaySeconds = selectDelay.div(web3.utils.toBN(1000_000_000))
+              console.log(`Last light client block: ${lastBlock.inner_lite.height}`)
+              console.log(`Time left to make a decision: ${selectDelaySeconds.toString()} seconds`)
+              await sleep(1000) // Block creation time is approximately one second, is an additional cli argument needed here?
+              continue
+            } else {
+              console.log('Time to make a decision is over!')
+            }
```

Comment:
Let's add some useful logs here (make it as nested logs so it will be clear to read - i.e. starting with "-> ":
* Block size
* Block height

When the block selection is over, print the summary on:
* which block at which height is selected
* how many bytes/gas are saved due to the selection

This will allow showing some stats on how actually this approach is useful.